docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc

```diff
@@ -19,6 +19,7 @@ The environment variables are:
 | 10000000 (or 10 MB)
 | Specifies the initial chunk size of uploaded files in bytes.
 The client will dynamically adjust this size within the maximum and minimum bounds (see below).
+To disable chunking completely, set these four variables to 0: `OWNCLOUD_CHUNK_SIZE=0 OWNCLOUD_MAX_CHUNK_SIZE=0 OWNCLOUD_MIN_CHUNK_SIZE=0 OWNCLOUD_CHUNKING_NG=0`
```

Comment:
I think setting OWNCLOUD_CHUNK_SIZE=0 should be enough to disable chunking.
What makes you think you need to set all of this. Did you try it and OWNCLOUD_CHUNK_SIZE=0 was not enough?

Reply:
Not tried. I suggested all 4 of them, just to be sure. 
What should the user check, to learn if chunking is on or off?

Reply:
OWNCLOUD_CHUNK_SIZE=0 is the only required one that disable the chunking completely.

Setting `OWNCLOUD_MIN_CHUNK_SIZE`  or `OWNCLOUD_MAX_CHUNK_SIZE` to 0 would be a very bad idea as it would mean that the client will want to send an infinite amount of chunk of size 0. These are only used in the computation of the chunk size and not in the decision to use chunking or not.
Setting `OWNCLOUD_CHUNKING_NG` would just force a chunking algorithm but not decide wether or not use chunks.

So in other words, only `OWNCLOUD_CHUNK_SIZE=0` is required and should be documented.

Reply:
Suggestion has been incorporated